src/routes/Chat/component.js

```diff
@@ -165,6 +166,9 @@ export default class Chat extends Component {
 							autoFocus={true}
 						/>}
 					</div>
+					<div>
+						{(messages[messages.length - 1] ? messages[messages.length - 1].t === 'webRTC_call_started' || messages[messages.length - 1].t === 'jitsi_call_started' : messages[messages.length - 1]) ? <CallNotification rid={(messages[messages.length - 1])} /> : null}
```

Comment:
TBH, I didn't like what we're doing here. The logic for determining if we should show the call notification shouldn't be present in this component bcs its not the responsibility of this component. We should actually pass a boolean prop to this component which would indicate if the callNotification component should be visible or not. Also I think we should be storing this boolean prop value within the Store itself (src/store/index.js)

Reply:
@Deepak-learner I've resolved this, Please take a look at my solutions and try to understand the changes I've made